### PR TITLE
Create pontificia-universita-gregoriana-teologia.csl

### PR DESCRIPTION
**Pull request by Copterus90**

### CODE REVIEW

**Comment by bwiernik** [comment]
Thanks for the updates!

@Copterus90 Please stop opening new pull requests. Please see here for information on editing existing pull requests https://github.com/citation-style-language/styles/blob/master/CONTRIBUTING.md#4-update-your-pull-request

File: pontificia-universita-gregoriana-teologia.csl [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+      <term name="ibid">ibid.</term>
+      <term name="in">in</term>
+      <term name="accessed">accesso</term>
+      <term name="editor">ed.</term>
+      <term name="editor" form="short">ed.</term>
+      <term name="page-range-delimiter">-</term>
+      <term name="volume">Vol. </term>
+      <!-- usamos "volume" para indicar el volumen en nota -->
+      <term name="ordinal-01"> vol.</term>
+      <!-- usamos "ordinal-01" para indicar el volumen en bibliografia (faltan terms!!) -->
```
Comment (bwiernik) [inline pontificia-universita-gregoriana-teologia.csl:33]: Can you explain the difference between what you want for the volume terms in the note versus bibliography? They look like the same word to me but different spaces? Hacking the ordinal term isn’t appropriate for that. What are you trying to do exactly?

File: pontificia-universita-gregoriana-teologia.csl [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+      <term name="ordinal-07">Vol. </term>
+      <!-- usamos "ordinal-07" para indicar el volumen en bibliografia (faltan terms!!) -->
+      <term name="from">orig. </term>
+      <term name="translator">, tr. </term>
```
Comment (bwiernik) [inline pontificia-universita-gregoriana-teologia.csl:37]: Including spacing and punctuation in terms like this isn’t appropriate. Please translate the term once. It looks like the default locales are the same, so these can just be removed. 

Spacing and punctuation should be controlled in the style using groups, delimiters, and prefixes/suffixes. Please take a read through  https://docs.citationstyles.org/en/stable/primer.html for discussion of best coding practices, including use of group, delimiter, and prefix/suffix

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks for your submission. It looks like you have embedded a lot of the style formatting (spacing, punctuation, etc) into term definitions. That’s not going to work. 

Please go through your style and use groups, delimiters, prefixes and suffixes, and attributes like text-case to control spacing, punctuation, and uppercasing. 

See the CSL Specification and CSL Primer for details. Please let me know if you have any questions. 

Please make all future edits to your style in this pull request rather than opening a new one.

File: pontificia-universita-gregoriana-teologia.csl (hunk 1 above)
Comment (Copterus90) [reply pontificia-universita-gregoriana-teologia.csl]: Solved with other terms and using in-line text formatting. Thanks for your advice

File: pontificia-universita-gregoriana-teologia.csl (hunk 2 above)
Comment (Copterus90) [reply pontificia-universita-gregoriana-teologia.csl]: Done

**Review by Copterus90 — COMMENTED** [review]
Changes in the manner to cite films and audios
Indented

**Comment by Copterus90** [comment]
> Thanks for your submission. It looks like you have embedded a lot of the style formatting (spacing, punctuation, etc) into term definitions. That’s not going to work.
> 
> Please go through your style and use groups, delimiters, prefixes and suffixes, and attributes like text-case to control spacing, punctuation, and uppercasing.
> 
> See the CSL Specification and CSL Primer for details. Please let me know if you have any questions.
> 
> Please make all future edits to your style in this pull request rather than opening a new one.

Requestes changes summited

File: pontificia-universita-gregoriana-teologia.csl (hunk 1 above)
Comment (bwiernik) [reply pontificia-universita-gregoriana-teologia.csl]: You control the casing of terms with thr `text-case` attribute, not by using other terms

File: pontificia-universita-gregoriana-teologia.csl [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+          <else-if type="entry-dictionary entry-encyclopedia paper-conference" match="any">
+            <text macro="ObraGeneral"/>
+          </else-if>
+          <!-- Libro -->
+          <else-if type="book chapter" match="any">
+            <choose>
+              <!-- Ebooks -->
+              <if variable="genre" match="any">
+                <group prefix=" [" suffix="]">
+                  <text term="preprint"/>
```
Comment (bwiernik) [inline pontificia-universita-gregoriana-teologia.csl:513]: This should just print the genre variable, not a term

File: pontificia-universita-gregoriana-teologia.csl [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+        <else>
+          <group>
+            <text macro="Autor_Nota" suffix=", "/>
+            <text macro="Titulo_Nota"/>
+            <choose>
+              <if type="book thesis" match="any">
+                <choose>
+                  <if variable="genre" match="any">
+                    <group prefix=" [" suffix="]">
+                      <text term="preprint"/>
```
Comment (bwiernik) [inline pontificia-universita-gregoriana-teologia.csl:447]: This should just print the genre variable, not a term

File: pontificia-universita-gregoriana-teologia.csl [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+      <term name="editor" form="short">ed.</term>
+      <term name="page-range-delimiter">-</term>
+      <term name="from">orig.</term>
+      <term name="translator">tr.</term>
+      <term name="volume">vol.</term>
+      <term name="review-of">Recensione di</term>
+      <term name="audio-recording">audio</term>
+      <term name="video">video</term>
+      <term name="internet">URL</term>
+      <term name="preprint">ebook</term>
```
Comment (bwiernik) [inline pontificia-universita-gregoriana-teologia.csl:39]: A preprint is not an ebook. Please remove this

File: pontificia-universita-gregoriana-teologia.csl [hunk 6]
```diff
@@ -0,0 +1,558 @@
...
+        <text term="volume" suffix=" " text-case="capitalize-first"/>
+        <text variable="volume" text-case="uppercase"/>
+      </else-if>
+    </choose>
+  </macro>
+  <!-- Formato de la colección -->
+  <macro name="Coleccion">
+    <group delimiter=" ">
+      <text variable="collection-title"/>
+      <text variable="collection-number" suffix=", "/>
```
Comment (anidal) [inline pontificia-universita-gregoriana-teologia.csl:197]:
```suggestion
    <group delimiter=" " suffix=", ">
      <text variable="collection-title"/>
      <text variable="collection-number"/>
```
Suffix after group, not inside! Cf. Norme Tipografiche p.27

File: pontificia-universita-gregoriana-teologia.csl [hunk 7]
```diff
@@ -0,0 +1,558 @@
...
+    </choose>
+  </macro>
+  <macro name="Volumen_Biblio">
+    <choose>
+      <if match="any" is-numeric="number-of-volumes">
+        <number text-case="uppercase" variable="number-of-volumes" form="numeric"/>
+        <text term="volume" prefix=" "/>
+      </if>
+      <else-if match="any" is-numeric="volume">
+        <text term="volume" suffix=" " text-case="capitalize-first"/>
```
Comment (anidal) [inline pontificia-universita-gregoriana-teologia.csl:188]:
```suggestion
  <macro name="Volumen_Nota">
    <choose>
      <if match="any" variable="volume">
        <text term="volume" text-case="capitalize-first" suffix=" "/>
        <text variable="volume" text-case="uppercase"/>
      </if>
    </choose>
  </macro>
  <macro name="Volumen_Biblio">
    <choose>
      <if match="any" is-numeric="number-of-volumes">
        <number text-case="uppercase" variable="number-of-volumes" form="numeric"/>
        <text term="volume" prefix=" "/>
      </if>
      <else-if match="any" variable="volume">
        <text term="volume" suffix=" " text-case="capitalize-first"/>
```

Volume may not be numeric, like in Norme Tipogrephiche p.28 line 759
Added also the space between vol. and number in footnote citation

**Comment by Copterus90** [comment]
> Thanks for your submission. It looks like you have embedded a lot of the style formatting (spacing, punctuation, etc) into term definitions. That’s not going to work.
> 
> Please go through your style and use groups, delimiters, prefixes and suffixes, and attributes like text-case to control spacing, punctuation, and uppercasing.
> 
> See the CSL Specification and CSL Primer for details. Please let me know if you have any questions.
> 
> Please make all future edits to your style in this pull request rather than opening a new one.

**Comment by Copterus90** [comment]
> Thanks for your submission. It looks like you have embedded a lot of the style formatting (spacing, punctuation, etc) into term definitions. That’s not going to work.
> 
> Please go through your style and use groups, delimiters, prefixes and suffixes, and attributes like text-case to control spacing, punctuation, and uppercasing.
> 
> See the CSL Specification and CSL Primer for details. Please let me know if you have any questions.
> 
> Please make all future edits to your style in this pull request rather than opening a new one.

A lot of changes are made on the code, and I think that all is formated in conformity with the rules of the CSL Specifications. When for another review? Thanks.

**Comment by adam3smith** [comment]
@bwiernik you'll finalize review here?

**Comment by bwiernik** [comment]
Yes

**Comment by Copterus90** [comment]
> @bwiernik you'll finalize review here?

Yes, the stile is finished

**Pull request closed without merging**